modules/flowable-json-converter/src/main/java/org/flowable/editor/language/json/converter/BaseBpmnJsonConverter.java

```diff
@@ -537,6 +537,9 @@ protected void addEventProperties(Event event, ObjectNode propertiesNode) {
 
             } else if (eventDefinition instanceof TimerEventDefinition) {
                 TimerEventDefinition timerDefinition = (TimerEventDefinition) eventDefinition;
+                if(StringUtils.isNotEmpty(timerDefinition.getCalendarName())) {
```

Comment:
please add spacing in `if(StringUtils.isNotEmpty` to match project standards; thanks.

Reply:
done!